packages/core/src/sass/app/_common/layout/modal-root.scss

```diff
@@ -8,7 +8,7 @@
     left: 0;
     bottom: 0;
     right: 0;
-    z-index: 9999;
+    z-index: 998;
```

Comment:
Why was this reduced?

Reply:
To fix the `MT5Dashboard` issue with `Popover` being rendered behind the `CompareAccounts` modal.